app/models/validations/sales/household_validations.rb

```diff
@@ -5,7 +5,7 @@ def validate_number_of_other_people_living_in_the_property(record)
     return if record.hholdcount.blank?
 
     unless record.hholdcount >= 0 && record.hholdcount <= 4
-      record.errors.add :hholdcount, I18n.t("validations.numeric.valid", field: "Number of other people living in the property", min: 0, max: 4)
+      record.errors.add :hholdcount, I18n.t("validations.numeric.within_range", field: "Number of other people living in the property", min: 0, max: 4)
     end
   end
 
```

Comment:
Can anyone see any reason why I should not remove this validation? I assume this was done before, or in innocent ignorance of, the validate_numeric_min_max method.

Reply:
fine by me, you'll need to add the min and max to the question object instead of course (and tests) but I agree that's cleaner